docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md

```diff
@@ -0,0 +1,37 @@
+# are_all_hvac_sys_fan_objs_autosized  
+
+**Description:** Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized. 
+
+**Inputs:**  
+- **U,P,or B-RMI**: The RMR in which the fan system object is defined. 
+- **hvac_sys_obj**: The hvac object in which the associated fan objects will be assessed.
+
+**Returns:**  
+- **are_all_hvac_sys_fan_objs_autosized**: Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized.   
+ 
+**Function Call:** 
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+
+## Logic:  
+- Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(RMI)`  
```

Comment:
This is just for one system, right? So the logic description should be to create dictonary of zones and terminal units associated with the HVAC system, correct?

Reply:
@mkarpman This dictionary has keys for multiple hvac systems. We extract the information for the hvac we care about at line 28 so I think the logic is correct.